src/api/app/components/notification_action_description_component.rb

```diff
@@ -28,6 +26,8 @@ def call
         "#{@user} made #{@recipient} #{@role} of #{@target_object}"
       when 'Event::RelationshipDelete'
         "#{@user} removed #{@recipient} as #{@role} of #{@target_object}"
+      when 'Event::BuildFail'
+        "Reason for the build: #{@notification.event_payload['reason']}"
```

Comment:
What's the significance of the reason?

Reply:
I'm going to address this in #13447

Reply:
See https://github.com/openSUSE/open-build-service/pull/13447/commits/cdfcd6674ec7b6c8540242f75e5214c2973802bc